components/detail/CustomerDetail.tsx

```diff
@@ -0,0 +1,31 @@
+import { Customer } from "@/lib/types/apiTypes"
+import { Box, Flex, Heading } from "@chakra-ui/layout"
+import React from "react"
+
+type CustomerDetailprops = {
+    customer: Customer
+}
+
+function CustomerDetail({ customer }: CustomerDetailprops): JSX.Element {
+    const { name, id, description, created, updated } = customer
+    return (
+        <Flex
+            flexDirection="column"
+            backgroundColor="white"
+            border="1px solid"
+            borderColor="gray.400"
+            borderRadius="0.2rem"
+            padding="1rem 1rem"
+        >
+            <Heading>
+                <Box>{name}</Box>
+            </Heading>
+            <Flex>Description: {description}</Flex>
+            <Flex>Id: {id}</Flex>
+            <Flex>Created: {created}</Flex>
```

Comment:
Currently these are ridiculously precise (like `2022-05-03T08:39:06.557445`). It would probably suffice to just convert these to dates.